src/crimson/net/SocketConnection.h

```diff
@@ -131,7 +128,14 @@ class SocketConnection : public Connection {
   static void discard_up_to(std::queue<MessageRef>*, seq_num_t);
 
   struct Keepalive {
-    ceph_timespec reply_stamp;
+    struct Req {
+      static constexpr char tag = CEPH_MSGR_TAG_KEEPALIVE2;
+      ceph_timespec stamp;
+    } __attribute__((packed)) req;
```

Comment:
static variables won't be part of the packed struct, will they?

Reply:
ahh, no, they won't.